yb-voyager/cmd/importData.go

```diff
@@ -951,11 +958,13 @@ func executeSqlFile(file string) {
 				fmt.Printf("\b \n    %s\n", err.Error())
 				fmt.Printf("    STATEMENT: %s\n", sqlStr[1])
 				if !target.ContinueOnError { //default case
-					fmt.Println(err)
-					os.Exit(1)
+					utils.ErrExit("%v", err)
 				}
 			}
+		} else { // restore the value in case of no error
+			retryCount = INDEX_RETRY_COUNT
```

Comment:
Once you have the separate function, you won't need such "arbitrary" variable updates...